src/lib.rs

```diff
@@ -1,4 +1,6 @@
-use std::{borrow::Cow, marker::PhantomData, sync::Arc};
+use log;
+use std::ffi::CString;
```

Comment:
please follow the existing style - add inside the `use std::{..}` that's already there